app/models/job.rb

```diff
@@ -1,5 +1,28 @@
 # frozen_string_literal: true
 
 class Job < ApplicationRecord
-  validates :company, :title, :description, :link, :location, :image_url, presence: true
+  SPONSORSHIP_LEVELS = { RUBY: 3, EMERALD: 2, SAPPHIRE: 1, OPAL: 0 }.freeze
+
+  validates :company,
+            :title,
+            :description,
+            :link,
+            :location,
+            :image_url,
+            :sponsorship_level,
+            presence: true
+
+  validates :sponsorship_level, inclusion: { in: SPONSORSHIP_LEVELS.values }
+
+  def as_json
+    {
+      id: id,
+      title: title,
+      company: company,
+      description: description,
+      location: location,
+      link: link,
+      image_url: image_url,
+    }
+  end
```

Comment:
I've implemented a custom `as_json` method here because I don't want to send `sponsorship_level` to the frontend. We may want to reveal on our site the levels of our sponsors, but I'd rather make that choice explicitly than accidentally expose it here. Also open to approaching this change differently.